web-components/src/components/tabs/Tabs.ts

```diff
@@ -315,6 +315,7 @@ export class Tabs extends ResizeMixin(RovingTabIndexMixin(LitElement)) {
 
       if (newIndex !== -1) {
         this.updateSelectedTab(newIndex);
+        console.log(newIndex);
```

Comment:
remove console.log

Reply:
Fixed

Reply:
I still see this here, maybe we're missing newer commits?